packages/extension/src/view/devtools/components/infoCard/learnMoreDropdown.tsx

```diff
@@ -0,0 +1,143 @@
+                    <p className="text-sm font-medium text-gray-900 truncate capitalize">
+                      Proposal
+                    </p>
+                    <a
+                      href={PSInfo[infoKey].proposal}
+                      className="text-xs text-blue-600 hover:text-blue-700"
+                      target="_blank"
+                      rel="noreferrer"
+                    >
+                      Public explanation for the proposed solution (Chrome)
+                    </a>
+                  </div>
+                </div>
+              </li>
+            )}
+            {PSInfo[infoKey].publicDiscussion && (
+              <li className="py-4">
+                <div className="flex items-center">
+                  <div className="flex-1 min-w-0">
+                    <p className="text-sm font-medium text-gray-900 truncate capitalize">
+                      Public Discussion
+                    </p>
+                    <a
+                      href={PSInfo[infoKey].publicDiscussion}
+                      className="text-xs text-blue-600 hover:text-blue-700"
+                      target="_blank"
+                      rel="noreferrer"
+                    >
+                      Public questions and feedback about the proposal
+                    </a>
+                  </div>
+                </div>
+              </li>
+            )}
+            {PSInfo[infoKey].videoOverview && (
+              <li className="py-4">
+                <div className="flex items-center">
+                  <div className="flex-1 min-w-0">
+                    <p className="text-sm font-medium text-gray-900 truncate capitalize">
+                      Video Overview
+                    </p>
+                    <a
+                      href={PSInfo[infoKey].videoOverview as string}
+                      className="text-xs text-blue-600 hover:text-blue-700"
+                      target="_blank"
+                      rel="noreferrer"
+                    >
+                      Short summary video
+                    </a>
+                  </div>
+                </div>
+              </li>
+            )}
+            {PSInfo[infoKey].devDocumentation && (
+              <li className="py-4">
+                <div className="flex items-center">
+                  <div className="flex-1 min-w-0">
+                    <p className="text-sm font-medium text-gray-900 truncate capitalize">
+                      Dev Documentation
+                    </p>
+                    <a
+                      href={PSInfo[infoKey].devDocumentation}
+                      className="text-xs text-blue-600 hover:text-blue-700"
+                      target="_blank"
+                      rel="noreferrer"
+                    >
+                      Developer documentation
+                    </a>
+                  </div>
+                </div>
+              </li>
+            )}
+          </ul>
+        </div>
```

Comment:
Shouldn't we reduce the code repetition by keeping the repeated `li` in a function? Something like this

```js
  const renderLink = (label: string, url: string) => (
    <li className="py-4">
      <div className="flex items-center">
        <div className="flex-1 min-w-0">
          <p className="text-sm font-medium text-gray-900 truncate capitalize">
            {label}
          </p>
          <a
            href={url}
            className="text-xs text-blue-600 hover:text-blue-700"
            target="_blank"
            rel="noreferrer"
          >
            {label.toLowerCase()}
          </a>
        </div>
      </div>
    </li>
  );
```

Reply:
Yes, repetition can be reduced by passing three props inside the new component.
Right now the text inside the `para` tag and the text inside the `anchor` tag are different.
Say 
```
<div className="flex-1 min-w-0">
  <p className="text-sm font-medium text-gray-900 truncate capitalize">
    Proposal
  </p>
  <a
    href={url}
    className="text-xs text-blue-600 hover:text-blue-700"
    target="_blank"
    rel="noreferrer"
  >
   Public explanation for the proposed solution (Chrome)
  </a>
</div>       
```